### DISCUSSION TITLE
SSAO not working on Example website

### DISCUSSION THREAD

**Discussion by dshot92** [google/model-viewer#4680] [Q&A] [answered]
### Description

I was trying to make ssao work in my website.
Went to check the example on

[Model-Viewer Example](https://modelviewer.dev/examples/postprocessing/index.html)

and moving the slide with default mode does not bring out the ambient occlusion

### Browser Affected
- [x] Chrome
- [x] Edge
- [x] Firefox

### OS
- [x] Android
- [x] Linux
- [x] Windows

**Comment by elalish** [comment]
I'm not sure what slider you're talking about, but our SSAO example is working for me: https://modelviewer.dev/examples/postprocessing/index.html#combining-effects (check and uncheck the box). Can you share screenshots and your device and browser info?

**Reply by dshot92** [reply]
The seconds example has a slider for the ssao

**Reply by elalish** [reply]
No, it has an opacity slider. Which works if you choose e.g. Add for blend mode.

**Reply by dshot92** [reply]
Yeah, sorry, i dont know why i missed that.
I've made this on glitch.me

[SSAO sonic](https://glitch.com/edit/#!/glow-spangled-forest)

**Reply by dshot92** [reply]
It seems like it works if i import the script from the local folder

[Local Imports](https://glitch.com/edit/#!/glow-spangled-forest?path=index.html%3A20%3A8)

and does not when i use the cdn from the examples

[CDN imports](https://glitch.com/edit/#!/twisty-silver-brie?path=index.html%3A21%3A0)

![image](https://github.com/google/model-viewer/assets/21959185/6ab03027-b73e-4dce-b48e-263043224093)

chrome 121.0.6167.160
Linux-6.6.10-76060610-generic-x86_64-with-glibc2.35 64 Bits, X11 UI

But i believe at this point i am doing something wrong with the imports

**Reply by dshot92** [reply, marked as answer]
I tested by downloading the entire Example page and found this

```js
<script type="module" src="../../node_modules/@google/model-viewer/dist/model-viewer-module.min.js"></script>
```
It seems like the examples is not uysing the current version of the model-viewer that is on the cdn.

Tried by importing it in my website insted of using the normal cdn
```js
<script type="module" src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.4.0/model-viewer.min.js"></script>
```

and it work like expected.

EDIT:
RTFMed so hard right now.
In the setup part at the top therers the correct import for the module.
But i flew past it as i had already the model-viewer imported.
